browser/net/brave_ad_block_tp_network_delegate_helper.cc

```diff
@@ -148,13 +161,35 @@ class AdblockCnameResolveHostClient : public network::mojom::ResolveHostClient {
     // See https://crbug.com/872665
     optional_parameters->source = net::HostResolverSource::DNS;
```

Comment:
`ResolveHostParameters::secure_dns_mode_override` for `SECURE` mode is useless without specifying doh_server.
So we have to do it through `DnsConfigOverrides`